### PR TITLE
Defining a new resource directory to be used with doxygen master 1.10.0

### PR DESCRIPTION
**Pull request by albert-github**
Doxygen master 1.10.0 has support for the c++ concept of modules but as in the past doxygen used the term "modules" for the grouping of information (commands like `\defgroup`, `\ingroup` etc.) a new term for the later has been introduced: Topics. The introduction of the new term also has some influence on the generated files and file names. The consequence of this is that part of CGAL documentation (i.e. the "Reference Manual" item) didn't appear anymore in a.o. the treeview.
- creating a new setup
- adding the "topics" to the layout files as in the past the "modules" were.
- adjusting the "hack.js" where some manipulation of the TreeView is done and this had "modules" coded into it.
- adjusting the "header_packages.html" as this used the name "modules.js"

### CODE REVIEW

**Comment by albert-github** [comment]
Ready to be tested.

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0-Ic-106

File: Documentation/doc/resources/1.10.0/BaseDoxyfile.in [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+HTML_FOOTER            = ${CGAL_DOC_RESOURCE_DIR}/footer.html
+
+# Doxygen stores a couple of settings persistently in the browser (via e.g.
+# cookies). By default these settings apply to all HTML pages generated by
+# doxygen across all projects. The HTML_PROJECT_COOKIE tag can be used to store
+# the settings under a project specific key, such that the user preferences will
+# be stored separately.
+# This tag requires that the tag GENERATE_HTML is set to YES.
+
+HTML_PROJECT_COOKIE    = CGAL
```
Comment (sloriot) [inline Documentation/doc/resources/1.10.0/BaseDoxyfile.in:496]: Do we have the option to not have any cookie?

Comment (sloriot) [reply Documentation/doc/resources/1.10.0/BaseDoxyfile.in:496]: I know there used to be there, that just reminded me that we talked at some point to try to get rid of it (for compliance with European regulation).

Comment (albert-github) [reply Documentation/doc/resources/1.10.0/BaseDoxyfile.in:496]: Removing them would mean that when opening an new page (e.g. shift click a link) the behavior would become undefined and e.g. jump back to default settings.

**Pull request merged**